addons/l10n_it_edi_sdicoop/models/account_edi_format.py

```diff
@@ -150,7 +150,8 @@ def _l10n_it_post_invoices_step_1(self, invoices):
                 invoice.message_post(
                     body=(_("Invoices for PA are not managed by Odoo, you can download the document and send it on your own."))
                 )
-                to_return[invoice] = {'attachment': attachment}
+                to_return[invoice] = {'attachment': attachment,
+                                      'success': True}
```

Comment:
```suggestion
                to_return[invoice] = {'attachment': attachment, success': True}
```
It's not that long, IMO you can keep it on one line.